src/materials/nodes/PointsNodeMaterial.js

```diff
@@ -12,31 +22,30 @@ class PointsNodeMaterial extends NodeMaterial {
 
 	}
 
+	/**
+	 * Constructs a new points node material.
+	 *
+	 * @param {Object?} parameters - The configuration parameter.
+	 */
 	constructor( parameters ) {
 
 		super();
 
+		/**
+		 * This flag can be used for type testing.
+		 *
+		 * @type {Boolean}
+		 * @readonly
+		 * @default true
+		 */
 		this.isPointsNodeMaterial = true;
 
-		this.lights = false;
-		this.transparent = true;
-
-		this.sizeNode = null;
```

Comment:
I have removed this property since I think it's from a time where `InstancedPointsNodeMaterial` was not available yet and we assumed we have to implement size support in `PointsNodeMaterial`.

Reply:
I was doing a review on `InstancedPointsNodeMaterial`, with Nodes we wouldn't need variations for all this kind of case, and once we have Raycaster and BoundBox through GPU we will have an optimized support for procerural things. I'll try to show something in the next few weeks.